gulp/tasks/hassio-panel.js

```diff
@@ -37,8 +37,8 @@ async function buildHassioPanel(es6) {
   });
 
   return minifyStream(stream, es6)
-    .pipe(rename(`hassio-main-${es6 ? 'latest' : 'es5'}.html`))
-    .pipe(gulp.dest('build-temp'));
+    .pipe(rename('hassio-main.html'))
+    .pipe(gulp.dest(es6 ? 'build-temp' : 'build-temp-es5'));
```

Comment:
I believe this branch is mostly good to go, but I'm not really familiar with the hassio build process. I couldn't find where this is used. I'm not a hassio user myself.

Reply:
https://github.com/home-assistant/hassio-build/tree/master/build-scripts/frontend-build-env and then with this PR added https://github.com/home-assistant/hassio-build/pull/47

Reply:
So if we change this, I should update that PR